docs/verify/simulators.mdx

```diff
@@ -6,15 +6,15 @@ description: Overview of simulators
 
 IBM Quantum features a collection of high-performance simulators for prototyping quantum circuits and algorithms.
```

Comment:
I think this intro should include some clarification that we are talking about the IBM Quantum cloud based simulators, not the Qiskit built in simulator. This could be a note box and link out to this part of the API ref: https://docs.quantum-computing.ibm.com/api/qiskit/providers

Reply:
Is there a more specific link? From a quick scan I can't easily find info on the Qiskit simulator. Would this be the right link? https://docs.quantum-computing.ibm.com/api/qiskit/providers_basicaer

Reply:
ah yep thats the better url 👍